src/internal/getGPUVersion.ts

```diff
@@ -1,4 +1,4 @@
-export const getGPUVersion = (model: string) => {
+export function getGPUVersion(model: string) {
```

Comment:
Are the `const` to `function` changes stylistic or do they serve a purpose?

Reply:
> The primary difference between function declarations and function expressions is that declarations are hoisted to the top of the scope in which they are defined, which allows you to write code that uses the function before its declaration.

(I can revert if wanted)